WordPress/Jetpack/Classes/NUX/New Landing Screen/Views/InfiniteScrollerView.swift

```diff
@@ -98,8 +98,11 @@ class InfiniteScrollerView: UIScrollView {
     }
 
     /// Called on each `CADisplayLink` frame and sets the vertical content offset if a scroller delegate is set.
-    @objc private func step() {
+    @objc private func step(displayLink: CADisplayLink) {
+        let deviceFps = 1 / (displayLink.targetTimestamp - displayLink.timestamp)
+
         guard
+            deviceFps > 0,
```

Comment:
It may be nice to consider limiting the animation to only devices that could operate a X FPS (perhaps 30) or greater, but I don't know if it'll be a real world problem. I _believe_ "Low Power Mode" on iOS will drop to 30.